Workbooks/Workloads/SQL/Add monitoring virtual machine/Add monitoring virtual machine.workbook

```diff
@@ -331,7 +331,7 @@
     {
       "type": 1,
       "content": {
-        "json": "Please reference passwords stored as Key Vault secrets.",
+        "json": "Please reference passwords stored as [Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/general/overview) secrets.",
```

Comment:
It would be good to link to SQL Insights Key Vault recommendation link on how to reference passwords - https://docs.microsoft.com/en-us/azure/azure-monitor/insights/sql-insights-enable#store-monitoring-password-in-key-vault

Reply:
Additionally, can we also add details on networking for KV reference passwords? I like the banner of Important and stating the statements below. Can we do a similar thing in workbooks?

It will be something like:

! Important
1. It is highly recommended to reference passwords in Key Vault. For more information see  https://docs.microsoft.com/en-us/azure/azure-monitor/insights/sql-insights-enable#store-monitoring-password-in-key-vault 
2. Ensure that network and security configuration allows the monitoring VM to access Key Vault. For more information, see Access Azure Key Vault behind a firewall and Configure Azure Key Vault networking settings.

![image](https://user-images.githubusercontent.com/65616869/145137524-73b55ce9-b23e-44f0-82c2-6efbbe0be3df.png)

Reply:
Suggested message: "For a secure configuration, it is required to store the password as a Key Vault secret. [Grant access](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/tutorial-windows-vm-access-nonaad) to the managed identity of monitoring VM and [enable network connectivity](https://docs.microsoft.com/azure/key-vault/general/access-behind-firewall) from the monitoring VM to the key vault where the secret is stored".

Instead of hardcoding the above doc links which can change over time, it would be better to create and use https://go.microsoft.com links. This is how links to docs are handled in the portal as a general practice. You can create these links if you go to https://aka.ms and choose go.microsoft.com as the host. The security group alias for our team should be the owner of each link.

Reply:
Thanks @masree and @dimitri-furman for the feedback. I have updated the messages and links based on the feedback. Please let me know if there's any more concerns.